src/wam2layers/preprocessing/cmip.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from wam2layers.config import Config
+
+WAM2CMIP = {  # wam2layers name -> cmip name
```

Comment:
Should we adopt CMOR names as standard, as opposed to ERA5?

Reply:
We could, it seems more logical to use standard names. All names are only in the internal code anyway.
We don't exactly use ERA5 names either, so that's why I called it "wam2layers names"

I do really dislike the "evspsbl" name though.